broccoli/version.js

```diff
@@ -25,9 +25,11 @@ module.exports.VERSION = (() => {
   // * current "build type" or branch name (in CI this is generally not
   //   present, but it is very useful for local / testing builds)
   // * the sha for the commit
-  let packageVersion  = pkg.version;
+  let packageVersion = pkg.version;
   let sha = info.sha || '';
   let suffix = process.env.BUILD_TYPE || info.branch;
+  // * remove illegal non-alphanumeric characters from branch name.
+  suffix = suffix.replace(/[^a-zA-Z\d\s-]/, '-');
```

Comment:
@rwjblue The issue now is one I was having before where the `suffix` var is null. It passes tests locally, just on Travis it fails. Any suggestions?

Reply:
Ah, I see. 

Something like:

`suffix = suffix && suffix.replace(...)`

Should work I believe...

Reply:
@rwjblue Yup, all good now.